neptune/internal/execution/execution_context.py

```diff
@@ -137,10 +136,14 @@ def _run_aborting_thread(self, abort_callback):
         else:
             return
 
-        websocket_factory = ReconnectingWebsocketFactory(
-            backend=self._backend,
+        websocket_factory = self._backend.websockets_factory(
+            # pylint: disable=protected-access
+            project_uuid=self._experiment._project.internal_id,
             experiment_id=self._experiment.internal_id
         )
+        if not websocket_factory:
+            return
```

Comment:
maybe this should not return and instead do `if websocket_factory -> AbortingThread()`

Reply:
I don't get this comment.